ngraph_bridge/ngraph_utils.cc

```diff
@@ -347,7 +347,7 @@ Status NgraphSerialize(const std::string& file_name,
         "Passed a null pointer as ng function to serialize");
   }
   try {
-    ngraph::serialize(ng_function, json_indentation);
+    serialized = ngraph::serialize(ng_function, json_indentation);
```

Comment:
good catch! 👍

Reply:
So - how did it ever worked? Are there any unit tests to catch this?